app/helpers/session_helper.py

```diff
@@ -0,0 +1,94 @@
+
+from functools import wraps
+
+from flask import session, url_for, redirect, abort, make_response, jsonify, current_app
+
+from app.models.util import token as token_utils
+from app.models.util import user as user_utils
+
+from app.models import User, Token
+
+session_key = 'schedulr_token'
```

Comment:
Do we not want to use the one in the settings? Or is this a different key?

Reply:
This is just a key to store the token in the sessions dict.. in and of itself the content is irrelevant, as long as it's unique enough. Ill rename to lower ambiguity.

Reply:
Fixed